src/NuGet.Core/NuGet.Commands/PackagesLockFileBuilder.cs

```diff
@@ -57,11 +57,26 @@ public PackagesLockFile CreateNuGetLockFile(LockFile assetsFile)
                     var framework_dep = framework?.Dependencies.FirstOrDefault(
                         dep => StringComparer.OrdinalIgnoreCase.Equals(dep.Name, library.Name));
 
+                    CentralPackageVersion centralPackageVersion = null;
+                    bool libraryVersionIsCentrallyDefined = framework?.CentralPackageVersions.TryGetValue(library.Name, out centralPackageVersion) == true;
```

Comment:
you are not using this any more.

```suggestion
                    framework?.CentralPackageVersions.TryGetValue(library.Name, out centralPackageVersion);
```

Reply:
right - removed